prow/cmd/deck/static/tide.html

```diff
@@ -4,12 +4,12 @@
         <meta charset="UTF-8">
         <title>Tide Status</title>
         <link id="favicon" rel="icon" type="image/png" href="favicon.ico">
-        <link rel="stylesheet" type="text/css" href="style.css?reload-me-please=1513905648">
+        <link rel="stylesheet" type="text/css" href="style.css?reload-me-please=1516734418">
         <link rel="stylesheet" type="text/css" href="extensions/style.css">
         <link href="https://fonts.googleapis.com/css?family=Roboto:400,700" rel="stylesheet">
         <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
         <link rel="stylesheet" href="https://code.getmdl.io/1.3.0/material.indigo-pink.min.css">
-        <script type="text/javascript" src="tidescript.js?reload-me-please=1513905648"></script>
```

Comment:
@BenTheElder This doesn't seem to be needed?

Reply:
the browser can cache scripts too, since we don't actually use caching headers so I added the stamp. How else are we loading this script?

Reply:
you're seeing the updated one differently on localhost / doing hard reloads with the dev tools open than end users will see on other devices. Either we need to properly set caching headers and decide on a caching strategy or we should be stamping this too.

Reply:
Ah I figured that it was fine because `/tide.js?var=tideData` doesn't seem to be cached, but that is because we explicitly specify `no-cache` in the response header when handling requests to that path.
I'll add this back. We need this for each of the script files then (plugin-help-script.js and script.js)

Reply:
Fixed.

Reply:
`no-cache` isn't correct. [1] though this stamp thing is a bit of a quick hack.   

> Forces caches to submit the request to the origin server for validation before releasing a cached copy.

If we really don't want caching we need [2]:
```
Cache-Control: no-cache, no-store, must-revalidate
Pragma: no-cache
Expires: 0
```

[1] https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control
[2] https://stackoverflow.com/a/2068407

What I think we probably should do though is use `Last-Modified` (go file serving sets this) + `Cache-Control: public, must-revalidate, max-age=0`. We could also do even fancier caching with `ETag` and / or `Expires` but we probably don't need to, I was actually just about to submit a fix for this later today.

Reply:
writing a follow-up to fix caching in deck more generally